Updated vertical tab

MithunAdavelly opened the pull request.

vanitaj commented:
<img width="130" alt="Screen Shot 2019-10-07 at 3 51 44 PM" src="https://user-images.githubusercontent.com/6647022/66354674-769e0600-e91a-11e9-8d13-9ffb7b501495.png">
The active style is not matching with the zeplin styles.

vanitaj commented:
<img width="546" alt="Screen Shot 2019-10-07 at 3 55 00 PM" src="https://user-images.githubusercontent.com/6647022/66354812-d85e7000-e91a-11e9-9890-3769a1d12b5e.png">
Active Style for horizantal tab is not matching with the zeplin styles

vanitaj commented:
Changes required for vertical tabs - 
1) The hover should not have rounded corners on the right side.
2) Spacing between icon and label.

vanitaj commented on example/src/components/tabs/DarkModeTabs.jsx:
```diff
@@ -27,10 +27,38 @@ const styles = theme => ({
+    fontFamily: 'OpenSans',
+    fontStyle: 'normal',
+    fontStretch: 'normal',
+    LetterSpacing: 'normal'
```
Please use camelCase for LetterSpacing

MithunAdavelly replied:
Hi @vanitaj, fixed this and updated the PR over here with the latest commit. Thank you!

vanitaj commented:
Changes required for vertical tabs -

The hover should not have rounded corners on the right side.
Spacing between icon and label.

MithunAdavelly commented:
> Changes required for vertical tabs -
> 
> 1. The hover should not have rounded corners on the right side.
> 2. Spacing between icon and label.

Hi @vanitaj, fixed this and updated the PR over here with the latest commit. Thank you!

vanitaj commented on src/components/CustomVerticalTabs.jsx:
```diff
@@ -0,0 +1,42 @@
+  }
+}))(Tabs)
+
+const CustomVerticalTabs = ({ classes, children, ...other }) => (
```
Can you rename it to VerticalTabs.

MithunAdavelly replied:
Hi @vanitaj, Updated the PR over here with the fix for latest review comments including the name change of this file and the hover styling of the icons and their following labels. Thank you!
